app/scripts/modules/amazon/serverGroup/details/rollback/rollbackServerGroup.html

```diff
@@ -7,8 +7,8 @@
     </div>
     <div class="modal-body confirmation-modal">
       <div class="row">
-        <div class="col-sm-4 col-sm-offset-1">
-          <strong>Restore Server Group</strong>
+        <div class="col-sm-3 sm-label-right">
+          <strong>Restore to</strong>
```

Comment:
Can probably omit the `<strong>` wrapper - `sm-label-right` already adds weight, I think...